torch/utils/_cxx_pytree.py

```diff
@@ -243,6 +244,27 @@ def _private_register_pytree_node(
         )
 
 
+def treespec_leaf() -> TreeSpec:
+    return optree.treespec_leaf(none_is_leaf=True, namespace="torch")
+
+
+def treespec_tuple(iterable: Iterable[TreeSpec], /) -> TreeSpec:
+    return optree.treespec_tuple(iterable, none_is_leaf=True, namespace="torch")
+
+
+def treespec_dict(
+    mapping: Mapping[Any, TreeSpec] | Iterable[tuple[Any, TreeSpec]] = (),
```

Comment:
```suggestion
    mapping: Mapping[Any, TreeSpec] | Iterable[Tuple[Any, TreeSpec]] = (),
```
Tuple is already imported, so for consistency.